app/decorators/course_decorator.rb

```diff
@@ -234,4 +243,19 @@ def find_max(attribute)
 
     subject_attributes.compact.max.to_s
   end
+
+  def has_excluded_subject_name?
+    exclusions = [
+      /Drama/,
+      /Media Studies/,
+      /PE/,
+      /Physical/,
+    ]
+    # Course with more than one subject and a course name following the pattern "Foo with bar" are still eligible for bursaries
+    if /with/.match?(object.name)
```

Comment:
`with`

has nothing to do with has_excluded_`subject_name`

Reply:
I've renamed the method `has_excluded_course_name?`